.github/workflows/selene.yml

```
@@ -0,0 +1,21 @@
+name: Selene
+on:
+  pull_request: ~
```

I'd consider myself quite familiar with YAML, but had to read the docs for the use of `~` being equal to `null`.

Can we simplify this to
```suggestion
  pull_request:
```